docs/api/index.rst

```diff
@@ -11,12 +11,13 @@ API
 Changelog
 ---------
 
-1.13 (2016-10-20)
+1.13 (2016-10-27)
```

Comment:
It seems to me that 1.13 have been used already: https://github.com/Kinto/kinto/pull/891/files#diff-2c14b00f3393d6e7355fb480528f38e3R22